purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -0,0 +1,105 @@
+package com.revenuecat.purchases
+
+import android.content.Context
+import java.util.concurrent.ExecutorService
+
+open class PurchasesConfiguration(builder: Builder) {
```

Comment:
Originally I wasn't planning to copy the whole `PurchasesConfiguration`... But I didn't know how to have the `Builder` have a different constructor per flavor without duplicating everything like this :(

Reply:
I don't think kotlin allows to declare an inner class or a constructor as an extension :(